AndelaOSP/Andela-Socials
# chore(move scripts): Move scripts to single directory

#### What Does This PR Do?
- Creates bash_scripts directory
- Moves all bash scripts to the above directory
- Modifies scripts to run
- Updates README to reflect changes

#### Description Of Task To Be Completed
This task entailed moving all bash scripts from the root directory to a separate directory (`scripts`) to make the project more organized.

#### How can this be manually tested?
To test this a developer may try setting up the project from scratch using instructions from the README.md file or run the scripts individually and check for any bash related errors.

#### What are the relevant Pivotal Tracker Stories?
 [finished #158186607]
